fedbiomed/common/secagg_manager.py

```diff
@@ -13,8 +13,11 @@
 from tinydb import TinyDB, Query
 
 from fedbiomed.common.utils import raise_for_version_compatibility, __default_version__
-from fedbiomed.common.constants import ErrorNumbers, BiprimeType, SecaggElementTypes, \
+from fedbiomed.common.constants import (
+    ErrorNumbers,
+    SecaggElementTypes,
     __secagg_element_version__
+)
```

Comment:
In all the `fedbiomed.common.secagg_manager` module, we should probably update `experiment_id` type from `Union[str, None]` to `str` (`None` was existing only for the biprime case).

This means updating argument types, docstrings, plus a little bit of code in `_raise_error_incompatible_requested_entry`